example/index.js

```diff
@@ -1,5 +1,34 @@
 import { AppRegistry } from 'react-native';
 import App from './src/App';
+import HelloWorld from './src/JuspayTopView';
+import TopViewAttached from './src/JuspayTopViewAttached';
 import { name as appName } from './app.json';
+import { NativeModules, Platform } from 'react-native';
+
+const LINKING_ERROR =
+  `The package 'HyperAPIUtils' doesn't seem to be linked. Make sure: \n\n` +
+  Platform.select({ ios: "- You have run 'pod install'\n", default: '' }) +
+  '- You rebuilt the app after installing the package\n' +
+  '- You are not using Expo Go\n';
+
+const HyperSdkReact = NativeModules.HyperSdkReact
```

Comment:
this is the incorrect way of importing HyperSdkReact in example.

Can you import via 'hyper-sdk-react' package